src/PlotWindow.cpp

```diff
@@ -12,6 +12,21 @@
 #include <QLabel>
 #include <QMessageBox>
 
+class PlotView : public QChartView {
+ public:
+  using QChartView::QChartView;
+
+ protected:
+  void mouseReleaseEvent(QMouseEvent* mouseEvent) override {
+    if(mouseEvent->button() == Qt::RightButton) {
+      chart()->zoomReset();
```

Comment:
Very useful feature. I frequently had to close the plot window because I zoomed badly (out too much, into too much, want to see another region etc.)

But right mouse button was zoom out. How do I zoom out now? Especially as the plot does not have a grace margin but data goes right to the edge, I zoomed out once then then could select the correct region (given the limitation that you can only zoom inside the plot, not across the axes).